File: Store/Store.Web/Modules/Store/DocumentType/DocumentTypeColumns.cs

namespace Store.Store.Columns
{
    using Serenity.ComponentModel;
    using System;

    [ColumnsScript("Store.DocumentType")]
    [BasedOnRow(typeof(Entities.DocumentTypeRow), CheckNames = true)]
    public class DocumentTypeColumns
    {
        //[EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
        //public Int32 DocumentTypeId { get; set; }
        [Width(250), EditLink]
        public String DocumentName { get; set; }
    }
}

File: 2026/Company/Company.Web/Modules/ServerTypes/Sales/DocumentTypeColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { DocumentTypeRow } from "./DocumentTypeRow";

export interface DocumentTypeColumns {
    DocumentTypeId: Column<DocumentTypeRow>;
    DocumentName: Column<DocumentTypeRow>;
}

export class DocumentTypeColumns extends ColumnsBase<DocumentTypeRow> {
    static readonly columnsKey = 'Sales.DocumentType';
    static readonly Fields = fieldsProxy<DocumentTypeColumns>();
}